Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lighting/StandardLighting.azsli

```diff
@@ -8,6 +8,8 @@
 
 #pragma once
 
+#define ENABLE_TRANSMISSION     0
```

Comment:
You might need to also update the StandardPBR .shadervariantlist files in AtomSampleViewer to remove the transmission related shader options.

Reply:
Will do